app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Hello user with id:<%=@id%>posts</h1>
\ No newline at end of file
+<section>
+ <%@user.each do |usr|%>
+  <div class="user-info">
+    <div class="user-img">
+        <img src=<%=usr.photo%>>
+    </div>
```

Comment:
- [ ] From your controller, the instance variable `@user` is not an array so using the `.each` method on it returns an error
<img width="1415" alt="image" src="https://user-images.githubusercontent.com/37901469/219340166-e4834e69-4807-4ffb-bf29-951e410d5460.png">